src/main/java/com/criteo/vips/Vips.java

```diff
@@ -80,13 +80,17 @@ private static boolean tryLoadLibrariesFromJar() throws IOException {
     }
 
     private static boolean isWindows() {
-        return SystemName.indexOf("win") >= 0;
+        return SYSTEM_NAME.contains("win");
     }
 
     private static void loadLibraryFromJar(String name) throws IOException {
         String libName = System.mapLibraryName(name);
         File temp;
         try (InputStream in = Thread.currentThread().getContextClassLoader().getResourceAsStream(libName)) {
+            if (in == null) {
```

Comment:
Thanks, this is a nice check, the previous error message was a `NullReferenceException` IIRC. Can you make this change in a separate commit?

Reply:
Now in separate commit